### ISSUE TITLE
Outdated links in README.md

### ISSUE THREAD

**Issue by MarkCallow** [KhronosGroup/glTF-Compressor#23]
I think the link to "glTF Compressor" just above Credits and that to the "live demo" given under WebApp should be changed to https://github.khronos.org/glTF-Compressor-Release/. And "A live demo" should be changed to "The live version" since it is now released, as far as I know.

I'll be happy to update the README if someone confirms the links should be fixed. Is a direct edit okay or do you want a PR?

**Comment by javagl** [comment]
I agree that the link should not be the (current) `https://phasmatic3d.github.io/glTF-Sample-Viewer-KTX-demo/`, but https://github.khronos.org/glTF-Compressor-Release/ instead.

(I'd be fine with a direct edit for this, but am not an official maintainer, so maybe someone else wants to chime in...)
